src/bundle/Resources/views/locationview.html.twig

```diff
@@ -89,6 +89,28 @@
                     {{ 'actionbar.trash'|trans|desc('Send to Trash')}}
                 </button>
             {{ form_end(trashLocationForm) }}
+
+            {{
+                form_start(moveLocationForm, {
+                    "action": path("ez_hybrid_platform_ui_location_move_location",
+                    {'contentId': content.id, 'locationId': locationId})
+                })
+            }}
+                {{ form_widget(moveLocationForm.newParentLocationId) }}
+
+                {{
+                    form_widget(moveLocationForm.move, {
+                        'label': 'actionbar.move'|trans()|desc('Move'),
+                        'attr': {
+                            'class': 'ez-button ez-button-action ez-js-run-universal-discovery',
+                            'data-ud-title': 'actionbar.move.selectbox.title'|trans()|desc('Select the location where you want to move the location'),
+                            'data-ud-container': true,
+                            'data-ud-confirm-fill': '#' ~ moveLocationForm.newParentLocationId.vars.id,
+                            'data-ud-confirm-fill-with': 'location.id',
```

Comment:
Could you please add `'data-ud-confirm-label': 'Move'`?
reference: https://github.com/ezsystems/hybrid-platform-ui/pull/95/files#diff-89cf99eb2ca64e40720b4f0294ec5407R45